golem-component-service-base/src/repo/component.rs

```diff
@@ -209,7 +200,14 @@ impl ComponentRepo for DbComponentRepo<sqlx::Sqlite> {
         component_id: &Uuid,
     ) -> Result<Vec<ComponentRecord>, RepoError> {
         sqlx::query_as::<_, ComponentRecord>(
-            "SELECT components.namespace, components.name, component_versions.* FROM components JOIN component_versions ON components.component_id = component_versions.component_id WHERE components.namespace = $1 AND component_versions.component_id = $2"
+            r#"
+                SELECT c.namespace AS namespace, c.name AS name, c.component_id AS component_id,
```

Comment:
You don't need `t.name as name` - just `t.name` is enough.

Optional: I would suggest this code style:
```sql
select
    c.namespace,
    c.name,
    c.component_id,
    cv.version,
    cv.size,
    cv.user_component,
    cv.protected_component,
    cv.protector_version,
    cv.metadata
from components c
join component_versions cv on c.component_id = cv.component_id
where c.namespace = $1
  and c.component_id = $2
```
i.e. each column on a separate line (just like in a long case class creation), each join on a separate line, each `and` on a separate line.

Reply:
changed